[2024-03-01 09:11:35,753] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: meeting.merge_data 2024-03-01T09:11:27.498526+00:00 [queued]>
[2024-03-01 09:11:35,768] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: meeting.merge_data 2024-03-01T09:11:27.498526+00:00 [queued]>
[2024-03-01 09:11:35,769] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2024-03-01 09:11:35,769] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2024-03-01 09:11:35,769] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2024-03-01 09:11:35,778] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): merge_data> on 2024-03-01T09:11:27.498526+00:00
[2024-03-01 09:11:35,783] {standard_task_runner.py:52} INFO - Started process 4032 to run task
[2024-03-01 09:11:35,786] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'meeting', 'merge_data', '2024-03-01T09:11:27.498526+00:00', '--job-id', '169', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpwrx0w5xa', '--error-file', '/tmp/tmp9w69yxeq']
[2024-03-01 09:11:35,789] {standard_task_runner.py:77} INFO - Job 169: Subtask merge_data
[2024-03-01 09:11:35,827] {logging_mixin.py:109} INFO - Running <TaskInstance: meeting.merge_data 2024-03-01T09:11:27.498526+00:00 [running]> on host 9deaaf5c1807
[2024-03-01 09:11:35,877] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=meeting
AIRFLOW_CTX_TASK_ID=merge_data
AIRFLOW_CTX_EXECUTION_DATE=2024-03-01T09:11:27.498526+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-03-01T09:11:27.498526+00:00
[2024-03-01 09:11:35,899] {python.py:151} INFO - Done. Returned value was: None
[2024-03-01 09:11:35,908] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=meeting, task_id=merge_data, execution_date=20240301T091127, start_date=20240301T091135, end_date=20240301T091135
[2024-03-01 09:11:35,959] {local_task_job.py:151} INFO - Task exited with return code 0
[2024-03-01 09:11:35,980] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
